Tests/LibWeb/Text/input/SVG/empty-filter-crash.html

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
-<svg>
-    <filter id="empty"></filter>
-    <g filter="url(#empty)"/>
-</svg>
-<script src="../include.js"></script>
-<script>
-    // FIXME: This should be a crash test.
-    test(() => {
-        println("PASS (didn't crash)");
-    })
```

Comment:
woops didn't notice before, but shouldn't we also delete the expectation file?

Reply:
I also didn't notice :sweat_smile:. Deleted.